src/views/Bank.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+  <div>
+    <div
+      class="bank-page pt-100 pb-40"
+      style="background-color: #eaebf9"
+      :style="{ marginTop: `-${navbarHeight}px` }"
+    >
+      <v-container>
+        <v-row class="" justify="center">
+          <v-col cols="12" md="7" align-self="center" class="text-center">
+            <Breadcrumb
+              class="text-center"
+              large
+              currentPage="For Bank"
+              link="partnerships/for-banks"
+            />
+
+            <h3
+              class="black--text font-weight-medium text-h2 mb-10"
+              style="line-height: 75.65px"
+            >
+              Innovation Driven by Partnerships
+            </h3>
+            <p class="description text-body-main">
+              Using our AI system and community presence, we have been able to
+              penetrate the unbanked communities promoting financial inclusion
+              in those areas. We have combined breakthrough tech and machine
+              learning to enable Financial Institutions empower the unbanked
+              with Digital Credit solutions.
+            </p>
+          </v-col>
+        </v-row>
+      </v-container>
+    </div>
+    <section class="pt-100 pb-100" style="background-color: #9e8af6">
+      <v-container>
+        <v-row>
+          <v-col cols="12" md="6" lg="6"> </v-col>
+        </v-row>
+      </v-container>
+    </section>
+    <section class="offer pt-80 pb-2" style="">
+      <v-container>
+        <v-row class="pa-6" align="center">
+          <v-col cols="12" md="7" lg="8">
+            <h3
+              class="text-h2 font-weight-bold black--text"
+              style="max-width: 50%"
+            >
+              What We Offer Our Partners
+            </h3>
+          </v-col>
+          <v-col col="12" md="5" lg="4" class="pa-6" align-self="end">
+            <v-card
+              class="mx-auto pa-4 mb-12"
+              style="
+                background-color: #e9f2fe;
+                border-radius: 8px;
+                border: 1px solid blue;
+              "
+              v-for="(item, i) in offers"
+              :key="i"
+            >
+              <v-card-title class="text-h4 font-weight-600 mb-6">{{
+                item.title
+              }}</v-card-title>
+              <v-card-text class="text-body-main">
+                {{ item.content }}
+              </v-card-text>
+              <template v-if="item.lists">
```

Comment:
This component should be animated - Contact Onoriode for how the animation of this component should be.